[Hold] Platform Outlook Source Connector docs (#277)
Unstructured-IO/docs: snippets/general-shared-text/outlook-platform.mdx

> Fill in the following fields:

Reviewer: @ds-filipknefel Please confirm that my edits are technically correct.

- **Name** (_required_): A unique name for this connector.
- **User Email** (_required_): The Outlook email account address to access messages from.
- **Folders** (_required_): A comma-separated list of email folders to download messages from. To access messages from subfolders as well, check the **Recursive** box.

> - **Client ID** (_required_): The client ID of the Microsoft Entra ID application that has access to the Outlook email account address. Can be found in App registrations' **Overview** sections of your application.

Reviewer: ```suggestion
- **Client ID** (_required_): The client ID of the Microsoft Entra ID application that has access to the Outlook email account address. This client ID can be found in the app registration's **Overview** section in the Azure portal.
```

- **Client Credential** (_required_): The secret ID of the corresponding Entra ID application. Can be read from App registrations' **Cerificates & secrets** sections of your application **only** right after creation.
- **Authentication Provider URL**: The authentication token provider for the Entra ID application. If this field is left blank, `https://login.microsoftonline.com` is used by default.

> - **Tenant**: The ID or domain name that is associated with the Entra ID application. If this field is left blank, `common` is used by default. Can be found in App registrations' **Overview** sections of your application.

Reviewer: ```suggestion
- **Tenant**: The ID or domain name that is associated with the Entra ID application. If this field is left blank, `common` is used by default. This value can be found in the app registration's **Overview** section in the Azure portal.
```
